modules/migration-configuring-cors-3.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-configuring-cors-3_{context}']
+= Configuring cross-origin resource sharing on an {product-title} 3 cluster
+
+You must configure cross-origin resource sharing on the {product-title} 3 cluster to enable communication between the CAM tool on the target cluster and the source cluster's API server.
+
+.Procedure
+
+. Log in to the {product-title} 4 cluster.
+. Obtain your CAM tool URL:
++
+----
+$ oc get -n openshift-migration route/migration -o go-template='(?i}//{{ .spec.host }}(:|\z){{ println }}' | sed 's,\.,\\.,g'
+----
+
+. Log in to the {product-title} 3 cluster.
+. Add the CAM tool URL to the `corsAllowedOrigins` stanza in the `/etc/origin/master/master-config.yaml` configuration file:
++
+----
+corsAllowedOrigins:
+- (?i)//migration-openshift-migration\.apps\.cluster\.com(:|\z) <1>
```

Comment:
@eriknelson please try these instructions and verify CORS is working.

Marco and I hit issues following these steps when pasting the route as shown above.
When we removed the (?i) and (:|\z) we had success

Reply:
That's strange. I confirmed the instructions for [3.11](https://docs.openshift.com/container-platform/3.11/architecture/infrastructure_components/web_console.html#corsAllowedOrigins).

Reply:
@apinnick I went back and confirmed, the instructions are correct for how we specify to configure the CORS.

The issue Marco hit earlier today was that when we ran curl to verify we set the Origin to 
  migration-openshift-migration.apps.cluster.com
When it really should be Origin: https://migration-openshift-migration.apps.cluster.com

I'll add a comment lower in doc to get a little more explicit with what to set for the curl verification